UnitySDK/Assets/ML-Agents/Editor/Tests/Sensor/RayPerceptionSensorTests.cs

```diff
@@ -18,4 +20,190 @@ public void TestGetRayAngles()
             }
         }
     }
+
+    public class RayPerception3DTests
+    {
+        void SetupScene()
+        {
+            /*
+             *      C
+             *    S   S
+             *      @
+             * ^
```

Comment:
What is that comment? Is it useful?

Reply:
Added some more comments. It's a top-down view of the agents and the objects that are added to the scene; I think it makes some of the math easier to understand.